diagnostic.md

```diff
@@ -54,7 +54,7 @@ so much for us?
 // your response here
 ```
 
-What does typing `rails console` in a directory that houses a rails app do for
+What does typing `bin/rails dbconsole` in a directory that houses a rails app do for
```

Comment:
I thought so too and Antony said he prefers `bin/rails dbconsole` here https://github.com/ga-wdi-boston/rails-api-single-diagnostic/issues/17#issuecomment-278048650